static/templates/settings/alert_word_settings_item.hbs

```diff
@@ -1,5 +1,5 @@
 {{! Alert word in the settings page that can be removed }}
-
+{{#with alert_words}}
```

Comment:
`alert_word` may be a better variable here.

Reply:
@sahil839 , I don't think so, as here we do have multiple alert_words and we're running a `with` loop on `alert_words` and printing out the value of key `word` for them.

Reply:
`with` is not for a loop. And if it was, then you are contadicting your own statement that this file is used to render a single item. See https://handlebarsjs.com/guide/builtin-helpers.html#with

Reply:
ahh, i got it, i'm sorry, updated the PR with the changes.

Reply:
thanks :)